snap/validate_test.go

```diff
@@ -225,6 +227,9 @@ version: 1.0
 }
 
 func (s *ValidateSuite) TestIllegalHookName(c *C) {
```

Comment:
I don't even see why abc123 would be intrinsically invalid, btw. I'm probably missing something.